py/torch_tensorrt/dynamo/converters/aten_ops_converters.py

```diff
@@ -0,0 +1,303 @@
+import logging
+from typing import Dict, Sequence, Tuple, Union
+import torch
+from torch_tensorrt.fx.converters import acc_ops_converters
+from torch_tensorrt.dynamo import dynamo_tensorrt_converter
+from torch.fx.node import Argument, Target
+
+from torch_tensorrt.fx.types import TRTNetwork, TRTTensor
+from torch_tensorrt.dynamo.converters.converter_utils import SourceIR
+from torch_tensorrt.dynamo.converters.impl.elementwise import (
+    trunc_div,
+    rsqrt,
+    fmod,
+    rsub,
+    clamp,
+)
+from torch_tensorrt.dynamo.converters.impl.normalization import (
+    batch_norm,
+    layer_norm,
+    softmax,
+)
+from torch_tensorrt.fx.converters.impl import activation
+from torch_tensorrt.dynamo.converters.impl.squeeze import squeeze
+from torch_tensorrt.dynamo.converters.impl.select import select
+from torch_tensorrt.dynamo.converters.impl.slice import slice_op
+from torch_tensorrt.dynamo.converters.impl.matmul import matrix_multiply
+from torch_tensorrt.dynamo.converters.impl.condition import where
+from torch_tensorrt.dynamo.converters.impl.unsqueeze import unsqueeze
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+def or_none(args, i):
+    return args[i] if len(args) > i else None
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.batch_norm)
+def aten_ops_batch_norm(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return batch_norm(
+        network,
+        target,
+        SourceIR.ATEN,
+        name,
+        args[0],
+        args[1],
+        args[2],
+        args[3],
+        args[4],
+        args[5],
+        args[6],
+        args[7],
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.div.default)
+@dynamo_tensorrt_converter(torch.ops.aten.div.Tensor_mode)
+@dynamo_tensorrt_converter(torch.ops.aten.div.Tensor)
+def aten_ops_div(
+    network: TRTNetwork,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    kwargs_new = {
+        "input": args[0],
+        "other": args[1],
+    }
+    rounding_mode = kwargs.get("rounding_mode")
+    if rounding_mode is None:
+        return acc_ops_converters.acc_ops_div(network, target, None, kwargs_new, name)
```

Comment:
why are we calling out to acc_ops here?

Reply:
Since there was a change to this operator in the `trunc_div` PR #1869, we need to copy the entire converter to ensure the changes are reflected